src/devices/Pn532/README.md

```diff
@@ -182,8 +182,9 @@ It is possible to emulate any Type A, Type B and Felica cards.
 Communication support:
 - [X] HSU serial port: fully supported
 - [X] I2C: supported
-- [X] SPI: supported but using a specific chip select pin as well as using LSB with reverse bytes function rather than built in function. This is due to current limitation in the SPI implementations for Raspberry Pi. **Important**: when using Chip Select, it must be a pin which is different from the SPI defined one.
-- [ ] Hardware reset pin: This can be done with the user code.
+- [X] SPI: supported but using a specific chip select pin
```

Comment:
since you're changing this file you can also edit line 9: `## USage` => `## Usage`